src/components/elementView/elementTable.tsx

```diff
@@ -1005,20 +1006,34 @@ class ElementTable extends React.Component<
 
   renderPermissionsButton = (item: UserInfoObject) => {
     return (
-      <AnchorButton
-        className="button-table"
-        intent="primary"
-        icon="shield"
-        minimal
-        onClick={() => {
-          this.setState({
-            editUserFormOpen: true,
-            selected_userid: item.id,
-            isEditFormOpen: false,
-          });
-        }}
-        disabled={item.username === this.state.username}
-      />
+      <Tooltip
+        content={
+          item.username === this.state.username
+            ? "Cannot edit your own user permissions"
+            : "Cannot edit within change plan"
+        }
```

Comment:
if youre admin you can revoke admin permissions from yourself?

Reply:
are you saying you should be able to do this? I don't think you should be able to. and I think I have implemented it accordingly (¿?)

Reply:
wait ignore thsi comment not important lol